src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj

```diff
@@ -0,0 +1,161 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{A021EDFF-45C8-4DC2-BEF7-36E1B3B8CFE8}</ProjectGuid>
+    <RootNamespace>TestHostApp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformMinVersion>10.0.18362.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
+    <OutputSubDir>Tests\Data</OutputSubDir>
+    <UseWmXml>true</UseWmXml>
+    <ConfigurationType>Application</ConfigurationType>
+    <NoOutputRedirection>true</NoOutputRedirection>
+  </PropertyGroup>
+
+  <Import Project="$(SolutionDir)\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
+
+  <Import Project="$(OpenConsoleDir)\src\common.build.pre.props" />
+
+  <!-- This project is _heavily_ cribbed directly from the TAEF samples. In
+  order to avoid breaking this project, it's been left largely unmodified. The
+  only modifications are those near the bottom of the file:
+  * References to our dependent winrt projects (Connection, Settings, Control,
+    App)
+  * Manual copy steps to copy the actual test code (TerminalApp.LocalTests.dll)
+    and the dlls that TerminalConnection is dependent upon, but don't role up
+    here for whatever reason.
+  -->
+
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'">
+    <UseDebugLibraries>true</UseDebugLibraries>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
+    <UseDebugLibraries>false</UseDebugLibraries>
+    <WholeProgramOptimization>true</WholeProgramOptimization>
+    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
+  </PropertyGroup>
+
+  <PropertyGroup Label="UserMacros">
+    <GenerateAppxPackageOnBuild>false</GenerateAppxPackageOnBuild>

```

Comment:
doc this

Reply:
This ended up not doing anything (strange?)

Reply:
Ended up not doing anything, but it's still here?

Reply:
Yea I must have necro'd it while trying to fix the connection dlls. Yanking again.